code/modules/food_and_drink/snacks.dm

```diff
@@ -1179,12 +1179,18 @@
 /obj/item/reagent_containers/food/snacks/donut
 	name = "donut"
 	desc = "Goes great with Robust Coffee."
-	icon = 'icons/obj/foodNdrink/food_snacks.dmi'
-	icon_state = "donut1"
+	icon = 'icons/obj/foodNdrink/donuts.dmi'
+	icon_state = "base"
+	flags = FPRINT | TABLEPASS | NOSPLASH
+	appearance_flags = KEEP_TOGETHER
 	heal_amt = 1
-	initial_volume = 20
-	initial_reagents = "sugar"
+	initial_volume = 50
+	initial_reagents = list("sugar" = 20)
 	food_effects = list("food_energized")
+	var/can_add_frosting = TRUE
+	var/list/styles = list("icing", "sprinkles")
+	var/list/style_tags = list("icing" = "icing", "sprinkles" = "sprinkles")
```

Comment:
Wouldn't just one list suffice?

Reply:
This was more to differentiate between the icon name and what was displayed to the user originally, but you're right, that can be removed now.